### PR TITLE
Sql hm2

### PR DESCRIPTION
**Pull request by DadizhaO**

### CODE REVIEW

File: src/main/resources/sql/QuarryHM2.sql [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+SELECT
+    description,
+    COUNT(o.product)
+FROM
+    orders     o
+    INNER JOIN products   p ON p.product_id = o.product
+GROUP BY
+    p.description
+HAVING
+    COUNT(o.product) > 3;
```
Comment (mkharkhuta) [inline src/main/resources/sql/QuarryHM2.sql:329]: more accurate condition would be >= 3:)

File: src/main/resources/sql/QuarryHM2.sql [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+
+--9. Знайти продукти в яких буква "а" зустрічається більше 2-ух раз
+--I tried it, but it does not work. I do not know why.
+
+SELECT
+    *
+FROM
+    products p
+WHERE
+    REGEXP_LIKE ( p.description,'а{2,}' );
```
Comment (mkharkhuta) [inline src/main/resources/sql/QuarryHM2.sql:440]: does it work?

File: src/main/resources/sql/QuarryHM2.sql [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+
+select* from products where description like '%а%а%';
```
Comment (mkharkhuta) [inline src/main/resources/sql/QuarryHM2.sql:442]: this request also takes into account the appearance of two 'a'

**Review by mkharkhuta — COMMENTED** [review]
need a little corrections

**Review by SerhiiPasko — APPROVED** [review]
Read Masha's comments

**Pull request merged**